.github/workflows/container.reusable.yml

```diff
@@ -0,0 +1,91 @@
+name: Build container image
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: Name of the image to build
+        required: true
+        type: string
+      tag:
+        description: Tag of the image to build
+        required: true
+        type: string
+      load:
+        description: Whether to load the image after building it
+        default: false
+        required: false
+        type: boolean
+      push:
+        description: Whether to push the image after building it
+        default: false
+        required: false
+        type: boolean
+      latest:
+        description: Whether to also build (and push, if push == true) the `latest` tag
+        default: false
+        required: false
+        type: boolean
+      artifact:
+        description: Name of the artifact where the binaries have been uploaded
+        required: false
+        default: nri-kubernetes
+        type: string
+      platforms:
+        description: Comma-separated platforms to build for
+        required: false
+        default: linux/amd64,linux/arm64,linux/arm
+        type: string
+
+jobs:
+  container:
+    name: Build container image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Download all artifacts from build job
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact }}
+          path: bin
+
+      # It is common for branch names to have / characters, which cannot be used as container tags.
+      - name: Sanitize tag
+        id: tag
+        run: |
+          tag="${{ inputs.tag }}"
+          echo "::set-output name=name::${tag//\//-}"
+
+      - name: Build container images
+        run: |
+          load=""
+          if [[ "${{ inputs.load}}" == "true" ]]; then
+            load="--load"
+          fi
+
+          docker buildx build $load --platform=${{ inputs.platforms }} \
+            -t ${{ inputs.name }}:${{ steps.tag.outputs.name }} \
+            .
```

Comment:
Which is the point of this step? Also misleading step name.

Reply:
It is basically ported from the previous pipeline, but the idea is to have an step in which you can load the image out from the `buildx` container. This is required to run a security scan on it, for example.

That being said, it is perhaps better to just have the `--load` argument always present and just condition the job with `if: ${{ inputs.load }}`. WDYT?

Reply:
I leave it to you because I do not see the point of having to load other containers (without specifying the container or providing a `.tar.gz`) on the CI pipeline.

If you see it useful, go for it. But I don't see it used anywhere so I don't see a point in having a feature we are not using.